arviz/stats/diagnostics.py

```diff
@@ -859,22 +891,32 @@ def _conv_quantile(ary, prob):
 
 def _mcse_mean(ary):
     """Compute the Markov Chain mean error."""
+    _numba_flag = Numba.numba_flag
     ary = np.asarray(ary)
     if _not_valid(ary, shape_kwargs=dict(min_draws=4, min_chains=1)):
         return np.nan
     ess = _ess_mean(ary)
-    sd = np.std(ary, ddof=1)
+    if _numba_flag:
+        ary = np.ravel(ary)
+        sd = np.sqrt(svar(ary, ddof=1))
+    else:
+        sd = np.std(ary, ddof=1)
     mcse_mean_value = sd / np.sqrt(ess)
     return mcse_mean_value
 
 
 def _mcse_sd(ary):
     """Compute the Markov Chain sd error."""
+    _numba_flag = Numba.numba_flag
     ary = np.asarray(ary)
     if _not_valid(ary, shape_kwargs=dict(min_draws=4, min_chains=1)):
         return np.nan
     ess = _ess_sd(ary)
-    sd = np.std(ary, ddof=1)
+    if _numba_flag:
+        ary = np.ravel(ary)
+        sd = np.sqrt(svar(ary, ddof=1))
```

Comment:
Should there be ssd/sstd function?

Reply:
I guess we can use _sqr, as you suggested.